app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_part = command.split()
+    cp, source, target = command_part
+    if source == target:
+        print("Source and target are the same")
```

Comment:
why do u need this, if program will continue to work ? 
![image](https://github.com/mate-academy/py-copy-file/assets/103201010/9ffde09d-9c06-40a2-b9ba-7a8144a2e1ac)